Bio/Nexus/StandardData.py

```diff
@@ -71,9 +73,13 @@ def __init__(self, data):
                     coding_list["t"] = "uncer"
                     continue
                 elif coding in [")", "}"]:
-                    raise NexusError('Improper character "' + coding +
-                                     '" at position ' + pos +
-                                     " of a coding sequence.")
+                    raise NexusError(
+                        'Improper character "'
+                        + coding
+                        + '" at position '
+                        + pos
+                        + " of a coding sequence."
+                    )
```

Comment:
Not a black issue, but this could be rewritten to avoid the string concatenation, e.g. use ``%s`` formatting.

Reply:
The exact output of the message has changed (missing quotes around the problem character), but I don't think that is critical.